spaceship.zsh

```diff
@@ -309,6 +314,23 @@ spaceship_swift_version() {
   fi
 }
 
+# GOLANG
+# Show current version of golang.
+spaceship_golang_status() {
+  [[ $SPACESHIP_GOLANG_SHOW == false ]] && return
+
+  $(command go version > /dev/null 2>&1) || return
+
+  local go_version=$(go version)
+
+  # Do not show GOLANG prefix if prefixes are disabled
+  [[ ${SPACESHIP_PREFIX_SHOW} == true ]] && echo -n "%B${SPACESHIP_PREFIX_GOLANG}%b" || echo -n ' '
```

Comment:
As I understand, The version of Go will be shown always. If it is, then that's isn't a Spaceship's way. You need to find a way to check when it's needed to shown Go's version.